docs/pages/versions/v54.0.0/sdk/checkbox.mdx

```diff
@@ -70,6 +71,14 @@ const styles = StyleSheet.create({
 });
 ```
 
+An example of how `expo-checkbox` appears on Android and iOS is shown below:
+
+<ContentSpotlight
+  alt="Checkbox component on Android and iOS."
+  src="/static/images/sdk/checkbox/example.png"
```

Comment:
Could we also see the a checked version?
Also does this component look identical on iOS and Android?